crates/ecolor/src/color32.rs

```diff
@@ -1,6 +1,16 @@
-use crate::{
-    fast_round, gamma_u8_from_linear_f32, linear_f32_from_gamma_u8, linear_f32_from_linear_u8, Rgba,
-};
+use once_cell::sync::Lazy;
+
+use crate::{fast_round, linear_f32_from_linear_u8, Rgba};
+
+static FROM_UNMULTIPLIED_LUT: Lazy<[u8; 256 * 256]> = Lazy::new(|| {
+    use crate::{gamma_u8_from_linear_f32, linear_f32_from_gamma_u8};
+    core::array::from_fn(|i| {
+        let [value, alpha] = (i as u16).to_be_bytes();
```

Comment:
I doubt it will affect performance at all, but little-endian feels like a more natural choice (considering basically all CPUs are LE these days)